src/services/peopleSearch.ts

```diff
@@ -75,6 +77,32 @@ export type PeopleSearchQuery = {
   involvement: string;
 };
 
+const getRenamedDepartments = async () => {
+  const dep = await getDepartments();
+  dep.forEach((d, i) => {
```

Comment:
Something to think about, but NOT required...
Create a getNewName() method just like getOldName(), and have getRenamedDepartments call getNewName().  

Pro: The two conversions are very similar, so it's even easier to see that they reverse each other (and that could be easily tested with a unit test).  And the forEach would have a very simple body.

Con: It might be little more code.

Reply:
Just resolved the (office of the) case, thank you for pointing it out!

Th only thing about calling both get old name and get new name in the getRenamedDepartments is that we only want to get the old name after the function search is called, but the new name is needed before calling the search function.